src/sources/soundsourceflac.cpp

```diff
@@ -89,13 +89,16 @@ SoundSource::OpenResult SoundSourceFLAC::tryOpen(
         const OpenParams& /*config*/) {
     DEBUG_ASSERT(!m_file.isOpen());
     if (!m_file.open(QIODevice::ReadOnly)) {
-        kLogger.warning() << "Failed to open FLAC file:" << m_file.fileName();
+        kLogger.warning()
+                << "Failed to open FLAC file:"
+                << m_file.fileName();
```

Comment:
Can you explain why you split here? I actually don't care, but in terms of fixing our stile guide we may need an advice for the best practice.

Reply:
To get a consistent 8 character indentation independent of how long the log statement actually gets.

Reply:
We don't need more rules, we need less! I'm here for getting things done, not for memorizing and validating rules ;)

The code should be readable. That's the only rule I need for those individual layout choices.